### PR TITLE
Adding some tests for BookReader.js

### PR DESCRIPTION
**Pull request by imskr**

### CODE REVIEW

File: tests/BookReader.test.js [hunk 1]
```diff
@@ -7,3 +10,51 @@ test('clamp function returns Math.min(Math.max(value, min), max)', () => {
...
+
+beforeEach(() => {
+  clock = sinon.useFakeTimers();
+});
+
+afterEach(() => {
+  clock.restore();
+});
+
+test('testing debounce', () => {
```
Comment (jbuckner) [inline tests/BookReader.test.js]: What are you testing in this test?

File: package.json [hunk 2]
```diff
@@ -20,7 +20,9 @@
   },
   "homepage": "https://github.com/internetarchive/bookreader#readme",
   "dependencies": {
-    "jquery": "^3.4.1"
+    "jquery": "^3.4.1",
+    "lodash": "^4.17.15",
+    "sinon": "^7.3.2"
```
Comment (jbuckner) [inline package.json]: Can `lodash` and `sinon` go in the `devDependencies` below?

Comment (imskr) [reply package.json]: Yes, i'll do it

File: tests/BookReader.test.js (hunk 1 above)
Comment (imskr) [reply tests/BookReader.test.js]: BookReader.util.debounce() function present in BookReader.js

Comment (imskr) [reply tests/BookReader.test.js]: I have missed the BookReader.util part.. I will add it.

**Pull request merged**